src/util/std_code.h

```diff
@@ -872,6 +872,36 @@ class code_function_callt:public codet
     op2().id(ID_arguments);
   }
 
+  explicit code_function_callt(const exprt &_function) : codet(ID_function_call)
```

Comment:
Mark the no-argument constructor as deprecated.

Reply:
done